patches/app-0002-301de6e.patch

```diff
@@ -0,0 +1,87 @@
+From 301de6eb25dda2b9342a006baadaa068c42b115e Mon Sep 17 00:00:00 2001
+From: Denis Chenu <denis@sondages.pro>
+Date: Wed, 15 Feb 2017 01:59:12 +0100
+Subject: [PATCH] [feature][yunohost] Activate a plugin by command - Needed for
+ update
+
+---
+ application/commands/ActivatePluginCommand.php | 70 ++++++++++++++++++++++++++
```

Comment:
To move DB direct access system to ls system (with a new ynh plugin (i work on it). Think we can manage really more easily ls via ynh after.